gists/remote-state-upgrade/README.md

```diff
@@ -0,0 +1,56 @@
+# Okctl remote state upgrade
+
+## Description
+
+This upgrade moves the `state.db` file from a local machine to an S3 bucket.
+
+## Installation
+
+### MacOs
+Download [remote-state-upgrader_v0.0.0_Darwin_amd64.tar.gz](dist/remote-state-upgrader_v0.0.0_Darwin_amd64.tar.gz) to
+your machine and run 
+```shell
+tar xvf remote-state-upgrader_v0.0.0_Darwin_amd64.tar.gz
+```
+
+Run the following and verify that the checksum matches 
```

Comment:
"Run the following" what? 
Should probably add `md5 remote-state-upgrader_v.0.0.0_Darwin_amd64.tar.gz`